internal/website/content/howto/secrets/crypt.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Encrypt & Decrypt Data"
+date: 2019-03-21T17:44:28-07:00
+draft: true
+weight: 2
+---
+
+Once you have [opened a secrets keeper][] for the secrets provider you want,
+you can encrypt and decrypt small messages using the keeper.
+
+[opened a secrets keeper]: {{< ref "./open-keeper.md" >}}
+
+## Encrypting data
+
+To encrypt data with a keeper, you call `Encrypt` with the byte slice you
+want to encrypt.
+
+```go
+plainText := []byte("Secrets secrets...")
+cipherText, err := keeper.Encrypt(ctx, plainText)
+if err != nil {
+    return err
+}
+```
+
+## Decrypting data
+
+To decrypt data with a keeper, you call `Decrypt` with the byte slice you
+want to decrypt. This should be data that you obtained from a previous call
+to `Encrypt` with a keeper that uses the same secret material (e.g. two AWS
+KMS keepers created with the same customer master key ID). The `Decrypt`
+method will return an error if the input data did not come from the same
```

Comment:
recap from private chat: It is definitely the case for `localsecrets` via [x/crypto/nacl/secretbox](https://godoc.org/golang.org/x/crypto/nacl/secretbox) and `awskms` via their [KMS Cryptographic Details](https://d1.awsstatic.com/whitepapers/KMS-Cryptographic-Details.pdf) white paper that decryption will fail if the ciphertext was not encrypted with the key you're attempting to decrypt it with.

Reply:
It would surprise me to learn that any of the managed services helpfully supply "you're using the wrong key" as an error, but I don't have a citation for that.